package.json

```diff
@@ -21,20 +21,18 @@
 	"main": "./dist/index.js",
 	"types": "./dist/index.d.ts",
 	"scripts": {
-		"build": "npm run clean && ./node_modules/.bin/tsc",
-		"clean": "./node_modules/.bin/rimraf ./dist",
+		"build": "npm run clean && npx tsc",
+		"clean": "npx rimraf ./dist",
 		"prepublishOnly": "npm run test && npm run build",
-		"touch-test-env": "touch .env.test",
+		"touch-test-env": "npx touch .env.test",
 		"test": "npm run touch-test-env && ts-node ./__tests__/e2e.test.js"
```

Comment:
I noticed that this doesn't actually make the tests work as it seems to want some login credentials in the `.env.test` file? Maybe something to sort out for a build?